vncdotool/command.py

```diff
@@ -257,30 +256,45 @@ def build_proxy(options: optparse.Values) -> VNCLoggingServerFactory:
 def add_standard_options(parser: optparse.OptionParser) -> optparse.OptionParser:
     parser.disable_interspersed_args()
 
-    parser.add_option('-p', '--password', action='store', metavar='PASSWORD',
-        help='use password to access server')
-
-    parser.add_option('-u', '--username', action='store', metavar='USERNAME',
-        help='use username to access server')
-
-    parser.add_option('-s', '--server', action='store', metavar='SERVER',
+    parser.add_option(
+        '-p',
+        '--password',
+        help='use password to access server',
+    )
+    parser.add_option(
+        '-u',
+        '--username',
+        help='use username to access server',
+    )
+    parser.add_option(
+        '-s',
+        '--server',
         default='127.0.0.1',
-        help='connect to VNC server at ADDRESS[:DISPLAY|::PORT] [%default]')
-
-    parser.add_option('--logfile', action='store', metavar='FILE',
-        help='output logging information to FILE')
-
-    parser.add_option('-v', '--verbose', action='count', default=0,
-        help='increase verbosity, use multiple times')
-
+        help='connect to VNC server at ADDRESS[:DISPLAY|::PORT] [%default]',
+    )
+    parser.add_option(
+        '--logfile',
+        metavar='FILE',
+        help='output logging information to FILE',
+    )
+    parser.add_option(
+        '-v',
+        '--verbose',
+        action='count',
+        default=0,
+        help='increase verbosity, use multiple times',
+    )
     return parser
 
 
 def setup_logging(options: optparse.Values) -> None:
     # route Twisted log messages via stdlib logging
     if options.logfile:
-        handler = logging.handlers.RotatingFileHandler(options.logfile,
-                                      maxBytes=5*1024*1024, backupCount=5)
+        handler = logging.handlers.RotatingFileHandler(
+            options.logfile,
+            maxBytes=5 << 20,
```

Comment:
nit, I find N*1024*1024 more readable, as I don't have to think as much about how big the number is.  Maybe we add a comment for size?

Reply:
Okay, I'll try to remember this for the next time.